Amazon Linux (Uyuni Only) (#867)
uyuni-project/uyuni-docs: modules/client-configuration/pages/clients-amazon.adoc

[[clients-amazon]]
= Registering {amazonlinux} Clients

This section contains information about registering traditional and Salt clients running {amazonlinux} operating systems.

ifeval::[{suma-content} == true]
[IMPORTANT]
====
{amazonlinux} repository URLs are available from {scc}, but the packages and metadata are provided by Amazon, not by {suse}.
====
endif::[]

[IMPORTANT]
====
Traditional clients are not available on {amazonlinux}{nbsp}2.
{amazonlinux}{nbsp}2 clients are only supported as Salt clients.
====

[WARNING]
====

> Amazon Linux instance always have the same `machine-id` id at `/etc/machine-id`

Reviewer: ```suggestion
{amazonlinux} instances always have the same `machine-id` id at `/etc/machine-id`.
```

> Make sure you regenerated the `machine-id` after the instance was created following the procedure at xref:administration:tshoot-registerclones.adoc[]

Reviewer: ```suggestion
Make sure you regenerate the `machine-id` after the instance is created.
For more information, see xref:administration:tshoot-registerclones.adoc[].
```
